### PR TITLE
Adding hexfusion's recommendations

### LINKED ISSUE #455: Add hexfusion's go to queries to our metrics.yaml
Sam recommends we should add

```
sum(rate(container_fs_writes_total[5m])) by (device, container, node) != 0
```

and

```
sum(rate(etcd_object_counts{}[5m])) by (resource) != 0
```

And add these to our ectd panels.

### PR DESCRIPTION
**Pull request by jtaleric**
Adding the etcd guru's goto prom queries to our collection

Fixes #455

### CODE REVIEW

File: roles/kube-burner/files/metrics-aggregated.yaml [hunk 1]
```diff
@@ -163,3 +163,10 @@ metrics:
 
   - query: sum(kube_node_status_condition{status="true"}) by (condition)
     metricName: nodeStatus
+
+  - query: sum(rate(container_fs_writes_total[5m])) by (device, container, node)
```
Comment (rsevilla87) [inline roles/kube-burner/files/metrics-aggregated.yaml:167]: This query returns a bunch of timeseries. Roughtly number of containers * number of devices * number of nodes
This is a capture from a 75 node cluster.

>image

As the goal of this query is to measure the disk usage made by etcd we can use this query instead:
>`sum(rate(container_fs_writes_total{container="etcd", device!~".+dm.+"}[5m])) by (device, container, node) > 0`
image

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: roles/kube-burner/files/metrics-aggregated.yaml [hunk 2]
```diff
@@ -163,3 +163,10 @@ metrics:
...
+    metricName: containerDiskUsage
+
+  - query: sum(rate(etcd_object_counts{}[5m])) by (resource)
```
Comment (rsevilla87) [inline roles/kube-burner/files/metrics-aggregated.yaml:170]: Adding > 0 at the end will reduce the number of timeseries:
Before:
image

After:
image

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: roles/kube-burner/files/metrics-aggregated.yaml (hunk 1 above)
Comment (jtaleric) [reply roles/kube-burner/files/metrics-aggregated.yaml:167]: This isn't what we are looking for though...

We could limit the query to just the master, but just etcd isn't helpful. This is to look for other containers who are competing for the disk.

File: roles/kube-burner/files/metrics-aggregated.yaml (hunk 2 above)
Comment (jtaleric) [reply roles/kube-burner/files/metrics-aggregated.yaml:170]: ack, i'll add that.

File: roles/kube-burner/files/metrics-aggregated.yaml (hunk 1 above)
Comment (rsevilla87) [reply roles/kube-burner/files/metrics-aggregated.yaml:167]:
```
(sum(rate(container_fs_writes_total{device!~".+dm.+"}[5m])) by (device, container, node) and on (node) kube_node_role{role="master"}) >  0

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**